### PR TITLE
chore(PPDSC-2832): simplify percy regeneration script

### PR DESCRIPTION
**Pull request by JohnTParsons**
PPDSC-2832

**What**

Original brief was https://nidigitalsolutions.jira.com/browse/PPDSC-2832
BUT upgraded Percy client is returning [false negatives](https://percy.io/516bdf6f/newskit/builds/25354610/changed/1413957337?browser=edge&browser_ids=18%2C34%2C35&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1280&widths=375%2C1280) and the current client isn't, so I've the client upgrade from the PR and renamed it "Simplify Percy regeneration script"

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [x] Updated relevant documentation

**I have tested manually:**
- [x] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2832-upgrade-percy
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2832-upgrade-percy/storybook

**Review by LukeFinch — COMMENTED** [review]
Typo

File: docs/scripts.md [hunk 1]
```diff
@@ -51,10 +51,16 @@ Scripts should be named using the following naming convention:
 
 * `e2e:docs:ci` -> starts serve:docs and runs the cypress docs tests headlessly.
 
+Run Percy from main branch if you want to force the baseline images to be updated. The are the two commands to run (through yarn) after you have switched to main branch and done a git pull are shown below:
```
Comment (LukeFinch) [inline docs/scripts.md:54]: Typo "The are two commands"

**Review by LukeFinch — APPROVED** [review]

File: docs/scripts.md (hunk 1 above)
Comment (JohnTParsons) [reply docs/scripts.md:54]: Will change to "The two commands to run"

**Review by mstuartf — DISMISSED** [review]

**Review by LukeFinch — DISMISSED** [review]

**Review by LukeFinch — APPROVED** [review]

**Review by mstuartf — APPROVED** [review]

**Pull request merged**